apis/outlook/mail/code/pkg/global/global.go

```diff
@@ -0,0 +1,8 @@
+package global
+
+const CredentialEnv = "GPTSCRIPT_GRAPH_MICROSOFT_COM_BEARER_TOKEN"
+
+var (
+	ReadOnlyScopes = []string{"TODO"}
+	AllScopes      = []string{"TODO"}
```

Comment:
Do these need to be replaced with something else? If not, can we add a comment explaining?

Reply:
Whoops lol. Yeah this proves that we probably don't need to specify scopes. I think I'll add them in anyway, in case we for some reason change how we handle authn/z.

Reply:
Fixed